packages/logging-winston/system-test/logging-winston.js

```diff
@@ -68,34 +71,48 @@ describe('LoggingWinston', function() {
             util.inspect(testTimestamp)
           );
         }
-      }
+      },
+
+      {
+        args: [ new Error('forth') ],
+        level: 'error',
+        verify: function(entry) {
+          assert.ok(entry.data.startsWith('Error: forth'));
+        }
+      },
+
+      {
+        args: [ 'fifth message', new Error('fifth') ],
+        level: 'error',
+        verify: function(entry) {
+          assert.ok(entry.data.startsWith('fifth message: Error: fifth'));
```

Comment:
I don't see any references to `msg` (or `message`, if `msg` is not the right field) in these tests. Should this (and the previous element) read `entry.data.msg.startsWith('...')`?